src/components/App.js

```diff
@@ -0,0 +1,42 @@
+import Header from './Header';
+import React, { useState, useEffect } from 'react';
+import { fetchFolderData } from '../api';
+import Banner from './Banner';
+import CardContainer from './CardContainer';
+import 'normalize.css';
+import '../reset.css';
+import Footer from './Footer';
+
+function App() {
+  const [folderData, setFolderData] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = await fetchFolderData();
+        setFolderData(data);
+      } catch (error) {
+        console.error('Error fetching folderData:', error);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  if (!folderData) {
+    return <div>Loading...</div>;
+  }
+
+  const { folder } = folderData;
+  const { name, owner, links } = folder;
+
+  return (
+    <div>
+      <Header />
+      <Banner name={name} owner={owner} />
+      <CardContainer links={links} />
+      <Footer />
+    </div>
```

Comment:
div 태그에 스타일링을 적용한다던지 이런 경우가 아니라면 여기서는 불필요하게 래핑이 되어있다고 볼 수도 있는데요~ 이런 경우에는 Fragment로 사용해주셔도 좋을 것 같아요~
```suggestion
    <>
      <Header />
      <Banner name={name} owner={owner} />
      <CardContainer links={links} />
      <Footer />
    </>
```

Reply:
좋습니다~~